### PR TITLE
update domain file so that all actions work locally

### PR DESCRIPTION
**Pull request by erohmensing**
- adds utterances that were called in the actions file but were not available as templates
  - completes fix of changing `howwereyoubuilt` to `ask_howbuilt` everywhere

### CODE REVIEW

**Comment by akelad** [comment]
omg i can't believe that was called `howwereyoubuilt` 🤦‍♀️

File: domain.yml [hunk 1]
```diff
@@ -537,6 +547,8 @@ templates:
 actions:
   - utter_ask_faq_tutorials
   - utter_greet
+  - utter_greet_name
```
Comment (akelad) [inline domain.yml]: we'll leave this here, but for educational purposes: you don't actually need to add all templates if they're only called from an action and not used in the stories :D

Comment (erohmensing) [reply domain.yml]: good to know! You just mean in the list actions, right? That makes sense.

Comment (akelad) [reply domain.yml]: exactly

**Pull request merged**